go/common/compression/data_compression_service.go

```diff
@@ -8,7 +8,8 @@ import (
 )
 
 type DataCompressionService interface {
-	Compress(blob []byte) ([]byte, error)
+	CompressRollup(blob []byte) ([]byte, error)
```

Comment:
Probably worth a comment on these two methods to explain the reason for the difference, or maybe renaming to `CompressForRollup` or something to make it clear they don't just take in a rollup